app/controllers/api/v1/schedules_controller.rb

```diff
@@ -0,0 +1,51 @@
+class Api::V1::SchedulesController < ApplicationController
+  before_action :set_schedule, only: %i[ show edit update destroy ]
+
+  # GET /schedules or /schedules.json
+  def index
+    @schedules = Schedule.all
+
+    render json: @schedules
+  end
+
+  # GET schedules/1 or schedules/1.json
+  def show
+    render json: @schedules
+  end
+
+  # POST  /schedules or /schedules.json
+  def create
+    @schedules = Schedule.new(schedule_params)
+
+    if @schedules.save
+      render json: @schedules, status: :created
+    else
+      render json: @schedules.errors, status: :unprocessable_entity
+    end
+  end
+
+  # PATCH/PUT schedules/1 or schedules/1.json
+  def update
+    if @schedules.update(schedule_params)
+      render json: @schedules
```

Comment:
[OPTIONAL] consider adding successful update status as well, as seen with create. Makes actions consistent. Try this for all controllers